test/unit/number-formatting/format-spec.js

```diff
@@ -24,18 +24,22 @@ describe("NumberFormatting format", () => {
 			expect(formatNumberNoRounding(0, 4)).toEqual("0.0000");
 			expect(formatNumberNoRounding(1.1, 4)).toEqual("1.1000");
 			expect(formatNumberNoRounding(1.1212, 1)).toEqual("1.1212");
+			expect(formatNumberNoRounding(5e-7, 1)).toEqual("0.0000005");
 
 			expect(formatNumberNoRounding(-1.1212, 1)).toEqual("-1.1212");
 			expect(formatNumberNoRounding(-1.1, 4)).toEqual("-1.1000");
+			expect(formatNumberNoRounding(-5e-7, 1)).toEqual("-0.0000005");
```

Comment:
I ignored that side because it takes a big value to make it be converted so scientific notation. Something like `5e21`.

I can add some asserts just for the sake of it, but for price formatting reasons I don't think it is relevant.